ext/native-decls/LoadResourceFile.md

```diff
@@ -10,11 +10,11 @@ char* LOAD_RESOURCE_FILE(char* resourceName, char* fileName);
 
 Reads the contents of a text file in a specified resource.
 If executed on the client, this file has to be included in `files` in the resource manifest.
-Example: `local data = LoadResourceFile("devtools", "data.json")`
+Example: `local data = LoadResourceFile("devtools", "client/data.json")`
 
 ## Parameters
 * **resourceName**: The resource name.
-* **fileName**: The file in the resource.
+* **directory**: The path of the file in the resource.
```

Comment:
```suggestion
* **filePath**: The path to the file, relative to the specified resource
```
This would make more sense over directory, the parameters must also match the function declaration